### DISCUSSION TITLE
Laravel important change 8

### DISCUSSION THREAD

**Discussion by itsalifadaei** [laravel/framework#34424] [General]
Friends who install the version change to version 8 or the new app may run into this problem.
Release such an important change in the document is not mentioned
Don't be a carrot

https://github.com/laravel/laravel/commit/9cbc3819f7b1c268447996d347a1733aa68e16d7#diff-b467cc786635bd6945b17726282f1c64

https://github.com/laravel/laravel/commit/58a98efb86c0c0a819e333a52eccf2a7de652f15

**Comment by driesvints** [comment]
Did you read the upgrade guide?

![Screenshot 2020-09-21 at 12 42 57](https://user-images.githubusercontent.com/594614/93758039-027e7000-fc08-11ea-9c02-b4d80df8ca2a.png)
